lib/smoke_detector/version.rb

```diff
@@ -1,3 +1,3 @@
 module SmokeDetector
-  VERSION = "1.1.5"
+  VERSION = "1.1.6"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.